debug/examples/shop-consumables-list.json5

```diff
@@ -157,4 +157,16 @@
             ],
         },
     },
+    emptyList: {
+        shop: {
+            value: "There is nothing to buy",
+            position: { offsetX: 0, offsetY: 0 },
```

Comment:
do we need offsets

Reply:
yes

Reply:
are these centred anyway?

Reply:
`0, 0` is centred. The config here is for example/show the options.